src/button-dropdown/interfaces.ts

```diff
@@ -258,6 +258,7 @@ export interface ItemProps {
 
 export interface InternalItem extends ButtonDropdownProps.Item {
   badge?: boolean;
+  isCurrentPage?: boolean;
```

Comment:
nit: I think it worths adding a doc comment to the new isCurrentPage and fullWidth to explain the purpose.

Reply:
update: is isCurrentPage used? I think it was replaced by the isCurrentBreadcrumb, right?